client/src/Config.ts

```diff
@@ -34,3 +34,7 @@ export const setToken = (token: string | null) => {
   currentToken = token;
   api = createApi();
 };
+
+export const getGuestToken = () => {
+  return localStorage.getItem('rideToken');
```

Comment:
Why not call it `guestToken`?

Reply:
I thought that on the client side, it's better to refer to it as a ride token as it is connected to the ride and not to a specific user. But we can change it to "guestToken" to prevent confusion.